app/electron/main.ts

```diff
@@ -3,13 +3,16 @@ import { ChildProcessWithoutNullStreams, spawn } from 'child_process';
 import { app, BrowserWindow, ipcMain, Menu, MenuItem, screen } from 'electron';
 import { IpcMainEvent, MenuItemConstructorOptions } from 'electron/main';
 import log from 'electron-log';
+import Store from 'electron-store';
```

Comment:
The commit title references saving the app version but the important part of this commit is that it sends the release info to the renderer. So maybe reflect this in the title to make it a bit easier to understand.

Reply:
highlighted the important stuff first